hishel/_sync/_pool.py

```diff
@@ -134,6 +134,14 @@ def handle_request(self, request: Request) -> Response:
                 )
 
                 final_response.read()
+
+                if self._controller.is_cachable(request=request, response=final_response):
+                    metadata = Metadata(
```

Comment:
I dont' think we need to create a new metadata here. The transport and pool implementations should be almost identical, in transports we do not create new metadata here

Reply:
I did a simple copy of the storage logic few lines below, but you're absolutely right. I'm just wondering if we should (in both cases transport or pool) increment number of uses or not there, or amend the metadata in any way before re-store.

Reply:
I guess it's already handled by create_hishel_response method

Reply:
I mean in the case of not a 304 (cached==False)